src/translations/installer_es.ts

```diff
@@ -0,0 +1,245 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="es_ES">
+<context>
+    <name>WindowsInstaller</name>
+    <message>
+        <source>Lang_es</source>
+        <translatorcomment>By Roberto Planchuelo Tejeiro</translatorcomment>
```

Comment:
```suggestion
        <source>__LANGUAGE_NAME__</source>
        <translatorcomment>This must be the name of one of the NSIS translation files at https://sourceforge.net/p/nsis/code/HEAD/tree/NSIS/trunk/Contrib/Language%20files/ without the file extension.</translatorcomment>
```
You must never change the `source` element. And please don't change the comment, it is here to help explain what the translation of this must be. You can add your name to the translators file, so it will show up in the translators dialog in the Birdtray `About` tab. See [the wiki page](https://github.com/gyunaev/birdtray/wiki/Add-a-new-translation#translators-file) for instructions.

Reply:
<source>__LANGUAGE_NAME__</source>
<translatorcomment>Spanish</translatorcomment>

Reply:
> __LANGUAGE_NAME__ Spanish

Sorry, what do you mean by that?

Reply:
> I think that the editor changes my comment, it deletes "translatorcomment" and "source"

What editor are you using? You should be able to just use the `Commit suggestion` button here on GitHub for this change.